WordPress/wordpress-playground
# Explore: Setup SQLite database integration without creating wp-content/db.php

## What is this PR doing?

Related to https://github.com/WordPress/wordpress-playground/discussions/1379

Loads the SQLite database integration without creating a drop-in `wp-content/db.php` plugin. Instead, it leans on the `auto_prepend_file` PHP option to provide a fake global `$wpdb` variable. This prevents WordPress from trying to connect to MySQL. The first time that `$wpdb` is used for reading or a method call, it loads the actual SQLite database integration.

## Follow-up work

Use the same SQLite setup method in the web version of Playground. This could happen either when a non-minified WordPress build is used or, for simplicity, always if we remove the SQLite integration plugin from the minified build.

## Testing instructions

Run `bun packages/playground/cli/src/cli.ts server --login` and confirm the server starts without any database errors.

Test coverage coming as a part of the larger boot protocol discussion, see https://github.com/WordPress/wordpress-playground/discussions/1379.
